Cargo.toml

```diff
@@ -41,7 +42,9 @@ log-error = []
 log-none = []
 
 nrf = ["nrf-hal-common"]
-lpc55 = ["lpc55-hal"]
+lpc55 = ["lpc55-03"]
+lpc55-03 = ["dep:lpc55-hal"]
+lpc55-04 = ["dep:lpc55-hal-04"]
```

Comment:
I prefer something like `lpc55-v0.3` and `lpc55-v0.4` to avoid any confusion

Reply:
Changed.